### ISSUE TITLE
Accessibility Checker should flag aria-disabled on anchor with href

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#358]
Steps:
- go to https://w3c.github.io/html-aria/tests/ahref-aria-disabled.html
- run Checker
- Violation should be flagged for the Test 1 anchor (that has href and aria-disabled="true")

**Comment by tombrunet** [comment]
If the link is `tabindex="-1"` with some events to swallow clicks, wouldn't this be allowed? We wouldn't be able to detect the events, but we might still want to ignore this if the tabindex makes the link not tabbable?

**Comment by tombrunet** [comment]
@carmacleod See my question above.

**Comment by tombrunet** [comment]
For example:
```
<a href="https://w3c.github.io/html-aria/#el-a" aria-disabled="true" tabindex="-1" onclick="return false;">a with href - not really disabled</a>
```

**Comment by aliunwala** [comment]
Triage meeing:

If we do this this, we might only want to trigger if it is tab-able

**Comment by marcjohlic** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Equal-Access-61ae71f9f8b4ec001e136462/issues/IBMa/equal-access/358?planning-poker) with ZenHub @aliunwala @drjoho @mbgower @philljenkins @shunguoy @tombrunet

**Comment by shunguoy** [comment]
this is covered in #721, close this as fixed
